### ISSUE TITLE
Values in Sector drop down 

### ISSUE THREAD

**Issue by mariel-oar** [open-apparel-registry/open-apparel-registry#2213]
## Overview 

This ticket specifies which values should be displayed in the sector drop down on the search / filter panel. 

### Solution 
1) Only sector values that have been submitted by a contributor should be displayed in the drop down list. 

### Additional Context 
Justin had previously shared these comments about a possible solution (also in #2187) --

I would recommend creating a separate issue for changing to the way the sector drop-down is populated.

- Add a contributors parameter to GET /api/sector/ that mirrors the contributor
- Update the logic of GET /api/sector
       - If there is no contributors parameter restore the previous behavior of getting sector values from the facility index
       - If there is a contributors parameter, get distinct sector values from FacilityLIstItemss submitted by the specified contributor
- Update the client code that makes the sectors API request to pass the contributors parameter if the page is loaded in embed mode.

**Comment by TaiWilkin** [comment]
In Staging, there are some older sector values from before we started saving non-valid sectors as product types, which makes the dropdown look a little messy. This shouldn't be an issue in Production, where all of the data will have gone through the updated processing. 
![Screen Shot 2022-10-11 at 10.17.58 AM.png](https://images.zenhubusercontent.com/5e41c4f13da639155d2ee456/48443361-dd20-4852-a6c3-bc7abaddff68)
